build.sbt

```diff
@@ -25,7 +25,7 @@ val elephantbirdVersion = "4.15"
 val hadoopLzoVersion = "0.4.19"
```

Comment:
Yeah I was going to follow up with a PR to set mima against 0.17.0 and also enable it in the `run_test.sh` script